specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,35 @@
+# Pricing Unit
```

Comment:
I really like the alignment to Unit Format for consistency. My only request would be to also add PricingBlockSize as the unit without the block size isn't enough information to understand how pricing works.

Reply:
Block size can be a fast follow in v1.x since it's only necessary in a small number of (still important) use cases. Added here: [FOCUS Data Model](https://docs.google.com/spreadsheets/d/148Hdg1ws1Co3MDI65T_KtGCAzbtmFbzOU0pGvRnmbq8/edit#gid=1318248972) as 1.x

Reply:
One of the notes in the Discussion/Scratch space refers to some suggestions for 1.0+